docs/tutorials/timeseries/advanced/forecasting-custom-model.ipynb

```diff
@@ -134,26 +134,13 @@
     "        # to define a custom preprocessing logic\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #80.            # Default hyperparameters for the model
</pre><p>Nit: This comment seems to be redundant, we already have a docstring above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/autogluon/autogluon/pull/5038/discussion/#comment-2042534608'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>